App/views.py

```diff
@@ -0,0 +1,24 @@
+from django.http import HttpResponse
+from django.shortcuts import render
+import json
+import requests
+
+def index(request):
+    return HttpResponse("Hello, world. You're at the polls index.")    
+
+def main(request):
+	if(request.method=='POST'):
+		usr = request.POST['username']  #Here, we get the useranme fetched from html page
+		URL = "https://en.wikipedia.org/w/api.php?action=query&format=json&list=usercontribs&ucuser="
+		URL = URL + usr
+		r = requests.get(URL)  #Recieved data in json-format
+		lst = json.loads(r.text)
```

Comment:
The request might fail (e.g. the API is down and responds with HTTP 503), in which case this won't work.

Reply:
Ok. Will handle this, when response code is other than 200.

Reply:
Done.